CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ The format is based on [Keep a Changelog][Keep a Changelog] and this project adh
 
 - Refactor : crate/module document move to pure Markdown ([#136](https://github.com/SpringQL/SpringQL/pull/136))
 
+### Added
```

Comment:
Could you use `### For developers` section?

refs: https://github.com/SpringQL/SpringQL/pull/154/files